packages/venia-concept/src/components/ForgotPassword/api.js

```diff
@@ -0,0 +1,4 @@
+export const resetPasswordRequest = () => {
+    // TODO: Handle reset password.
+    return Promise.resolve();
```

Comment:
Is there not a Magento API endpoint for this?

Reply:
According to the description for the story, it is not a part of it and should be implemented separately.